lib/modules/manager/gradle/artifacts.ts

```diff
@@ -96,7 +99,10 @@ export async function updateArtifacts({
 
   const fileList = await scm.getFileList();
   const lockFiles = fileList.filter((file) => isLockFile(file));
-  if (!lockFiles.length) {
+  const verificationMetadataFile = fileList.find((fileName) =>
+    fileName.endsWith('verification-metadata.xml')
```

Comment:
```suggestion
    fileName.endsWith('gradle/verification-metadata.xml')
```
to not find any file which ends with same name